beacon-chain/state/setters.go

```diff
@@ -39,6 +40,7 @@ const (
 	previousJustifiedCheckpoint
 	currentJustifiedCheckpoint
 	finalizedCheckpoint
+	validatorIdxMap
```

Comment:
Please add comment that this isn’t a field used by ssz

Reply:
Alright, done